third_party/terraform/tests/resource_dataflow_job_test.go

```diff
@@ -128,6 +168,70 @@ func testAccDataflowJobExists(n string) resource.TestCheckFunc {
 	}
 }
 
+func testAccDataflowJobHasNetwork(n string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("No ID is set")
+		}
+
+		config := testAccProvider.Meta().(*Config)
+
+		job, err := config.clientDataflow.Projects.Jobs.Get(config.Project, rs.Primary.ID).Do()
+		if err != nil {
+			return fmt.Errorf("Job does not exist")
+		}
+		resource.Retry(1*time.Minute, func() *resource.RetryError {
```

Comment:
I think both of these retry fns are redundant? The service account test these look like they were modeled after made a network call: https://github.com/GoogleCloudPlatform/magic-modules/blob/7bfb74bc582a335abdbf22e50b95a60be5a7f4fa/third_party/terraform/tests/resource_dataflow_job_test.go#L256-L257

Judging by the fact that only entries with no templates were retried, that API is probably delayed / eventually consistent. Here, we're just examining the object we retrieved so we can return an error without a retry.

Reply:
Oh, yeah, that's pretty silly. I meant to put the network call in the retry block. I think it's not necessary, anyway.

Reply:
Turns out it is necessary. I moved the network call in the retry block.